EIPS/eip-3779.md

```diff
@@ -234,11 +233,10 @@ func validate_path(pc := 0, sp := 0, bp := 0, rp := 0) int, error {
          // pop jump destination
          jumpdest = data_stack[--sp]
          if !valid_jumpdest(jumpdest) {
-            return return pc, invalid_destination
-
-
+            return pc, invalid_destination
          }
-         continue;
+         pc = jumpdest
```

Comment:
This is the only change that I believe fixes the bug in algorithm, please check if I got the intent right @gcolvin 

(Also it could be `jumpdest + 1`, because JUMPDEST is noop anyway)